docs/AwsMarketplaceGuide.md

```diff
@@ -1,12 +1,14 @@
 # How to install Scalar products through AWS Marketplace
 
-Scalar products (ScalarDB, ScalarDL, and their tools) are available in the AWS Marketplace as container images. This guide explains how to install Scalar products through the AWS Marketplace.
+Scalar products (ScalarDB Server, ScalarDB Cluster, ScalarDL, and their tools) are available in the AWS Marketplace as container images. This guide explains how to install Scalar products through the AWS Marketplace.
```

Comment:
```suggestion
Scalar products (ScalarDB, ScalarDL, and their tools) are available in the AWS Marketplace as container images. This guide explains how to install Scalar products through the AWS Marketplace.
```
For here, I think it's probably better and consistent to just say ScalarDB and ScalarDL.